asm/macros/battle_script.inc

```diff
@@ -1674,6 +1674,12 @@
 	callnative BS_FickleBeamDamageCalculation
 	.endm
 
+	.macro jumpifallyblocksoundmove battler:req, failInstr:req
```

Comment:
Since this can receive any battler, I would rename it to `jumpifblocksoundmove`.